Mathlib/Analysis/Matrix.lean

```diff
@@ -482,6 +497,13 @@ theorem frobeniusIsBoundedSMul [SeminormedRing R] [SeminormedAddCommGroup α] [M
     IsBoundedSMul R (Matrix m n α) :=
   (by infer_instance : IsBoundedSMul R (PiLp 2 fun i : m => PiLp 2 fun j : n => α))
 
+/-- This applies to the frobenius norm. -/
```

Comment:
IMHO it would be more polite to capitalize `Frobenius`, it is a personal name after all; but that's kind of orthogonal to this PR since the lowercase version is used in this file already.

Reply:
I'll do another PR with this change throughout.

Reply:
#25122

Reply:
```suggestion
/-- This applies to the Frobenius norm. -/
```